### PR TITLE
Update ChatRoom context type docs title

### PR DESCRIPTION
**Pull request by bingenito**
Document title for ChatRoom context type was incorrect

---
THIS SOFTWARE IS CONTRIBUTED SUBJECT TO THE TERMS OF THE FINOS CORPORATE CONTRIBUTOR LICENSE AGREEMENT.
 
THIS SOFTWARE IS LICENSED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE AND ANY WARRANTY OF NON-INFRINGEMENT, ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. THIS SOFTWARE MAY BE REDISTRIBUTED TO OTHERS ONLY BY EFFECTIVELY USING THIS OR ANOTHER EQUIVALENT DISCLAIMER IN ADDITION TO ANY OTHER REQUIRED LICENSE TERMS.

### CODE REVIEW

**Comment by bingenito** [comment]
This can be closed/canceled if the context generated docs is merged as that version does not have the issue.

**Comment by kriswest** [comment]
@bingenito Also needs fixing here (which docs generation won't fix):
- https://github.com/finos/FDC3/blob/e56b674735f0989df11c8fac6c540091e805b889/website/versioned_docs/version-2.1/context/ref/ChatRoom.md?plain=1#L4

**Comment by bingenito** [comment]
Updated versioned 2.1 md

File: website/versioned_docs/version-2.1/context/ref/ChatRoom.md
Comment (kriswest) [inline website/versioned_docs/version-2.1/context/ref/ChatRoom.md:4]:
```suggestion
title: Chat Room
```

File: website/versioned_docs/version-2.1/context/ref/ChatRoom.md
Comment (kriswest) [inline website/versioned_docs/version-2.1/context/ref/ChatRoom.md:3]:
```suggestion
sidebar_label: Chat Room
```

File: docs/context/ref/ChatRoom.md
Comment (kriswest) [inline docs/context/ref/ChatRoom.md:4]:
```suggestion
title: ChatRoom
```

File: website/versioned_docs/version-2.1/context/ref/ChatRoom.md
Comment (kriswest) [inline website/versioned_docs/version-2.1/context/ref/ChatRoom.md:4]:
```suggestion
title: ChatRoom
```

**Review by kriswest — DISMISSED** [review]
LGTM

File: docs/context/ref/ChatRoom.md
Comment (kriswest) [inline docs/context/ref/ChatRoom.md:3]:
```suggestion
sidebar_label: ChatRoom
```

**Review by kriswest — DISMISSED** [review]
LGTM again (after I messed around with it inexpertly)

**Comment by bingenito** [comment]
@kriswest   Note that plder doc generation is consistent on pascal case and no spaces across all context type markdown.  The new docs have the spacing.  This is now no longer consistent with other pages.

**Comment by kriswest** [comment]
I see what you mean, generated docs:
image

Thats coming from the titles in the Schemas - I can raise something to match those up. I think its jus tthe Chat types that are affected.

**Comment by kriswest** [comment]
We'll need another reviewer here as I touched this branch last and therefore no longer count due to the updated governance settings. @finos/fdc3-desktop-agent-maintainers

**Comment by kriswest** [comment]
> @kriswest Note that old docs are consistent on pascal case and no spaces across all context type markdown. The new docs have the spacing. This is now no longer consistent with other pages.

fixed in #1285

**Review by kriswest — APPROVED** [review]
LGTM

**Comment by bingenito** [comment]
It is disabled for me as I'm one of the PR authors.

**Review by openfin-johans — APPROVED** [review]

**Pull request merged**
